src/config/__jest__/Config.test.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs';
+import Config from '../Config';
+import lodash from 'lodash';
+
+jest.mock('fs');
+describe('Default variables', () => {
+  let testedConfig = new Config();
+  test('Default clientID and token are empty', () => {
+    expect(testedConfig.clientId).toEqual('');
+    expect(testedConfig.token).toEqual('');
+  });
+  test('Default language is English', () => {
+    expect(testedConfig.language).toEqual('en');
+  });
+  test('Default command prefix is !', () => {
+    expect(testedConfig.prefix).toEqual('!');
+  });
+  test('Default whitelisted extensions are mp3 and wav files', () => {
+    expect(testedConfig.acceptedExtensions).toEqual(['.mp3', '.wav']);
+  });
+  test('Default maximum filesize is ', () => {
+    expect(testedConfig.maximumFileSize).toEqual(1000000);
+  });
+  test('Default setting to delete messages is false', () => {
+    expect(testedConfig.deleteMessages).toBeFalsy();
+  });
+  test('Default setting to stay in the channel is false', () => {
+    expect(testedConfig.stayInChannel).toBeFalsy();
+  });
+  test('Default setting to deafen the bot is false', () => {
+    expect(testedConfig.deafen).toBeFalsy();
+  });
+  test('Default game is not set', () => {
+    expect(testedConfig.game).toEqual('');
+  });
+});
+
+describe('Setting config from Environment Variables', () => {
+  const OLD_ENVIRONMENT_VARIABLES = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...OLD_ENVIRONMENT_VARIABLES };
+  });
+
+  afterEach(() => {
+    process.env = OLD_ENVIRONMENT_VARIABLES;
+  });
+
+  let randomString = (length: number) => {
```

Comment:
Define this outside for improved readability?

Also, why do we need `lodash.times` here? As far as I understand, it would invoke the `times` times and return an array with all the results?

From the invocations below, I guess what we want is to create a random string with length `x` wrapped in an array because that is how things are used in `Config`? 🤔

Reply:
Replaced it with something simpler :)